app/views/demographic_questions/_form.html.erb

```diff
@@ -36,6 +36,11 @@
     <%= f.check_box :optional %>
   </div>
 
+  <div class="field">
```

Comment:
Don't forget to add this to the show page!

Reply:
It is done now!

Reply:
Thanks @spencerldixon